packages/firebase_storage/android/src/main/java/io/flutter/plugins/firebase/storage/FirebaseStoragePlugin.java

```diff
@@ -53,6 +53,24 @@ public void onMethodCall(MethodCall call, final Result result) {
     }
 
     switch (call.method) {
+      case "FirebaseStorage#getMaxDownloadRetryTime":
+        result.success(firebaseStorage.getMaxDownloadRetryTimeMillis());
+        break;
+      case "FirebaseStorage#getMaxUploadRetryTime":
+        result.success(firebaseStorage.getMaxUploadRetryTimeMillis());
+        break;
+      case "FirebaseStorage#getMaxOperationRetryTime":
+        result.success(firebaseStorage.getMaxOperationRetryTimeMillis());
+        break;
+      case "FirebaseStorage#setMaxDownloadRetryTime":
+        setMaxDownloadRetryTimeMillis(call, result);
+        break;
+      case "FirebaseStorage#setMaxUploadRetryTime":
+        setMaxUploadRetryTimeMillis(call, result);
+        break;
+      case "FirebaseStorage#setMaxOperationRetryTime":
+        setMaxOperationTimeMillis(call, result);
+        break;
```

Comment:
Alternatively, reduce to set/getRetryTime(maxDownload, maxUpload, maxOperation) to avoid a platform channel invocation per value.